pkg/controller/kogitoapp/builder/builder_test.go

```diff
@@ -47,8 +47,10 @@ func TestBuildResources_CreateAllWithoutImage(t *testing.T) {
 	s.AddKnownTypes(appsv1.SchemeGroupVersion, &appsv1.DeploymentConfig{}, &buildv1.BuildConfig{})
 	s.AddKnownTypes(imgv1.SchemeGroupVersion, &imgv1.ImageStreamTag{}, &imgv1.ImageStream{})
 
+	sa := NewServiceAccount(kogitoApp)
+
 	inv, err := BuildOrFetchObjects(&Context{
-		Client:    &client.Client{ControlCli: fake.NewFakeClient(), ImageCli: imgfake.NewSimpleClientset().ImageV1()},
+		Client:    &client.Client{ControlCli: fake.NewFakeClient(&sa), ImageCli: imgfake.NewSimpleClientset().ImageV1()},
```

Comment:
we can remove the SA references in the code completely. Only the SA name will be used and this can be set by a constant in the build package.

Reply:
Deleted